### ISSUE TITLE
Community Section - Slack

### ISSUE THREAD

**Issue by NikRimington** [umbraco/OurUmbraco#219]
I know there have been discussions regarding Slack, but considering it is a fairly well-established platform for the community (and other communities such as UmbBristol, etc). It could be worth at least having a nice link/post saying that it exists/advertised.

Gitter is fine (I've only just heard of it thanks to this community update) and I get that it's a HQ decision to include it with more of a feed but I believe that Slack still deserves to be promoted.

**Comment by nul800sebastiaan** [comment]
It is great that the Slack channel exists and some people are comfortable there. 

However, the goal of the HQ is to get a lively community going on Our Umbraco and so we don't want to send people somewhere else to create a shadow-community. 

We also don't like with the closed nature of the community Slack where all information basically disappears into a void due to the 10.000 message rollover. It's a shame that valuable information disappears in there to only be picked up by a few people.

In short: we considered integrating the Umbracian Slack and looked at it from multiple angles, the conclusion was that it doesn't make sense for the goals of growing the Our Umbraco community site.
